docs/adr/0019-use-1password-for-sharing-secrets.md

```diff
@@ -0,0 +1,96 @@
+# Use 1Password for sharing secrets
+
+* EASi engineers occasionally need access to secret information to test the app
+  locally. For example, API keys connect to CEDAR, or other services the app
+  depends upon. These secrets rotate periodically, sometimes outside of our
+  control, and sometimes without warning. This creates a challenge: how can we
+  share an updated secret in a secure way with minimal interruption to ongoing
+  app engineering work?
+
+* Decision drivers:
+  1. EASi engineers need to be able to stay up-to-date on secret values that
+     are required for app development.
+  1. There should be a central source of truth for the secrets.
+  1. The scheme for storing, sharing, and retrieving secrets should be
+     consistent with our understanding of best security practices.
+  1. The scheme should also be user-friendly and accessible without assuming
+     any knowledge of back-end infrastructure or overly complex authentication
+     mechanisms.
+
+## Considered Alternatives
+
+* Email
+* Slack DMs
+* 1Password
+* AWS SSM / ParameterStore
+
+## Decision Outcome
+
+Chosen Alternative: Use a 1Password vault for sharing secrets.
+
+* 1Password allows us to control access to the secrets repository on a per-user
+  basis, and updates to any secret will be available to other users on the
+  vault immediately.  With a separate vault within 1Password, we need not share
+  secrets with other Truss personnel who are not assigned to the project.
+* 1Password also has familiar and user-friendly graphical interfaces via the
+  web and a multi-platform desktop application.
+* Access to secrets can be revoked as quickly and as easily as it was granted.
+
+Consequences:
+
+* We will need to create a vault, which takes time and costs money.
+* Some synchronization will be required on how to use the vault and what
+  secrets to store in it.
+* Secrets that are required by the easi-backend app will have to be updated in
+  SSM anyway, which means changing them requires updating the value in two
+  places (1Password and SSM).
+
+## Pros and Cons of the Alternatives
+
+### Email
+
+* `+` Fast.
+* `+` Some of our peers share secrets with us using email, so forwarding the
+  emails is an easy and straightforward way to distribute secrets.
```

Comment:
```suggestion
```

Reply:
This doesn't seem right... Maybe with a comma? I dunno. What needs fixing here?

Reply:
I think we should keep the ADR focused to our team's processes.

Reply:
I don't think I'm reading github's suggestion highlighting right. Is this saying you want to delete the "and"?

Reply:
My github suggestion was to delete this second item entirely. Is it basically restating the first point?

Reply:
9aaa6e3aa704db238402fe868929db2acb2b0aef